### ISSUE TITLE
print traning_loss in the checkpoint filename

### ISSUE THREAD

**Issue by zhenyulin** [jcjohnson/torch-rnn#164]
As discussed in the char-nn repo document, the training_loss mark in the checkpoint filename is very useful for sampling.

Would it be possible to bring back this for more efficient exploration?

**Comment by dgcrouse** [comment]
Output dir is already selectable via -checkpoint_name option.

If you would like to add loss to a checkpoint name, feel free to develop it yourself and submit a pull request.

**Comment by GabiGrin** [comment]
@zhenyulin after having the exact same need I found this PR https://github.com/jcjohnson/torch-rnn/pull/181
Until merged, I'm manually changing the file. Should work for you too :)

**Comment by InnovativeInventor** [comment]
I think adding training loss to the file name should be a flag option. Otherwise it might mess up some scripts that depend on having nice checkpoint names.
